tests/src/JIT/HardwareIntrinsics/X86/Popcnt/Popcnt.cs

```diff
@@ -0,0 +1,131 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using System;
```

Comment:
It would be nice if we could just move this to be templated.

Reply:
This test will be changed again after exposing 64-bit intrinsic in nested classes. Let’s move them later.